lib/active_fedora/containers/container.rb

```diff
@@ -0,0 +1,28 @@
+module ActiveFedora
+  class Container < ActiveFedora::Base
+
+    property :membership_resource, predicate: ::RDF::Vocab::LDP.membershipResource
+    property :member_relation, predicate: ::RDF::Vocab::LDP.hasMemberRelation
+
+    def parent
+      @parent || raise("Parent hasn't been set on #{self.class}")
+    end
+
+    def parent=(parent)
+      @parent = parent
+      self.membership_resource = [::RDF::URI(parent.uri)]
+    end
+
+    def mint_id
+      "#{id}/#{SecureRandom.uuid}"
+    end
+
+    def self.find_or_initialize(id)
```

Comment:
@no-reply Specifically in relation to the original comment which said to move this method to AF::Base. Seems like keeping it just on containers keeps the AF::Base api smaller.